src/configure/configure.ts

```diff
@@ -408,98 +446,75 @@ class Orchestrator {
-        switch (this.inputs.pipelineConfiguration.template.targetType) {
-            case TargetResourceType.None:
-                break;
-            case TargetResourceType.WebApp:
-            default:
-                let appServiceClient = new AppServiceClient(this.inputs.azureSession.credentials, this.inputs.azureSession.environment, this.inputs.azureSession.tenantId, this.inputs.subscriptionId);
-                let selectedPipelineTemplate = this.inputs.pipelineConfiguration.template;
-                let matchingPipelineTemplates = templateHelper.getPipelineTemplatesForAllWebAppKind(this.inputs.sourceRepository.repositoryProvider,
-                    selectedPipelineTemplate.label, selectedPipelineTemplate.language, selectedPipelineTemplate.targetKind);
-
-                let webAppKinds = matchingPipelineTemplates.map((template) => template.targetKind);
-                let selectedResource: QuickPickItemWithData = await this.controlProvider.showQuickPick(
-                    Messages.selectTargetResource,
-                    appServiceClient.GetAppServices(webAppKinds)
-                        .then((webApps) => webApps.map(x => { return { label: x.name, data: x }; })),
-                    { placeHolder: Messages.selectTargetResource },
-                    TelemetryKeys.AzureResourceListCount);
-
-                if (await appServiceClient.isScmTypeSet((<GenericResource>selectedResource.data).id)) {
-                    this.continueOrchestration = false;
-                    await openBrowseExperience((<GenericResource>selectedResource.data).id);
-                }
-                else {
-                    this.inputs.targetResource.resource = selectedResource.data;
-                    this.inputs.pipelineConfiguration.template = matchingPipelineTemplates.find((template) => template.targetKind === <TargetKind>this.inputs.targetResource.resource.kind);
-                }
-        }
-    }
-
     private async getSelectedPipeline(): Promise<void> {
         const repoAnalysisHelper = new RepoAnalysisHelper(this.inputs.azureSession, this.inputs.githubPATToken);
         let repoAnalysisResult = null;
         if (this.inputs.sourceRepository.repositoryProvider === RepositoryProvider.Github) {
             repoAnalysisResult = await repoAnalysisHelper.getRepositoryAnalysis(this.inputs.sourceRepository, this.inputs.pipelineConfiguration.workingDirectory.split('\\').join('/'));
         }
-
         extensionVariables.templateServiceEnabled = false;
+        var appropriatePipelines: PipelineTemplate[] = [];
 
-        let appropriatePipelines;
-        // TO:DO- Get applicable pipelines for the repo type and azure target type if target already selected
-
-        if (extensionVariables.templateServiceEnabled) {
-            repoAnalysisResult = null;
-            appropriatePipelines = await vscode.window.withProgress(
+        if (!extensionVariables.templateServiceEnabled) {
```

Comment:
is this condition sufficient? In case of Azure repos, we would still want to use local repo analysis and local templates.

Reply:
or may be this is right. So if I understand right, analyzeRepoAndListAppropriatePipeline2() merges both local template and remote templates, and hence for azure repos, it still will be able to list local azure pipeline templates. Similarly for AKS. Correct?

Reply:
Yes, analyzeRepoAndListAppropriatePipeline2() merges both local and remote templates.

Reply:
I think analyzeRepoAndListAppropriatePipeline2 will return the github remote templates + Azure local templates in case of azure repos.

Reply:
Assuming that we'll be calling template service for github templates only, have added the check for repoProvider.